packages/autocannon-ui-frontend/src/components/RunOptionsForm.jsx

```diff
@@ -237,13 +259,128 @@ export default function RunOptionsForm(props) {
                       <HelpIcon color="primary" className={classes.helpIcon} />
                     </Tooltip>
                   ),
-                  inputProps: { min: 0 }
+                  inputProps: { min: 1 }
                 }}
                 value={options.duration}
                 onChange={e => onOptionChange('duration', e)}
                 fullWidth
               />
             </Grid>
+            <Grid item xs={12}>
+              <Container maxWidth="sm" className={classes.actionSection}>
+                <Box display="flex" justifyContent="center">
+                  <Grid
+                    container
+                    direction="column"
+                    justify="center"
+                    alignItems="center"
+                    spacing={3}
+                  >
+                    <Button
```

Comment:
button cannot be a child of a grid container. grid container's children must be grid items